src/server.rs

```diff
@@ -908,13 +911,15 @@ impl WebThingServer {
     ///   things
     /// * `port` - port to listen on (defaults to 80)
     /// * `hostname` - optional host name, i.e. mything.com
+    /// * `dns_service` - optional libmdns::Service name
     /// * `ssl_options` - tuple of SSL options to pass to the actix web server
     /// * `action_generator` - action generator struct
     /// * `base_path` - base URL to use, rather than '/'
     pub fn new(
         things: ThingsType,
         port: Option<u16>,
         hostname: Option<String>,
+        dns_service: Option<libmdns::Service>,
```

Comment:
What's the point of passing this parameter in if it's just overwritten on line 1130?

Reply:
Fixed: https://github.com/WebThingsIO/webthing-rust/pull/54/commits/dc9056247f5e904eccef600b30da5a4049a7bd0d